Cross.toml

```
@@ -3,3 +3,6 @@ passthrough = [
 	"RUSTFLAGS",
 	"RUST_TEST_THREADS"
 ]
+[target.loongarch64-unknown-linux-gnu]
+image = "ghcr.io/loongarchlinux/archlinux:latest"
```

Looks like this should be updated as well, sorry for missing it:

```suggestion
image.name = "ghcr.io/loongarchlinux/archlinux:latest"
```

---

Let me commit this to test it:)